package.json

```diff
@@ -108,7 +108,8 @@
     "ts-essentials": "^9.2.0",
     "tss-react": "^3.7.1",
     "validator": "^13.7.0",
-    "xlsx": "^0.18.5"
+    "xlsx": "^0.18.5",
+    "xss": "^1.0.14"
```

Comment:
`xss` was the smallest and tied for most-downloaded, and it's recently updated. 

https://npmtrends.com/dompurify-vs-sanitize-html-vs-xss
https://bundlephobia.com/package/xss@1.0.14